cypress/TestCases/FireboltCertification/DiscoveryLaunchHot.feature

```diff
@@ -211,4 +211,35 @@ Feature: Discovery.launch_HotLaunch
         Given the environment has been set up for 'DiscoveryLaunch' tests
         And '3rd party app' transitions to state 'foreground'
         When 1st party app invokes the 'Firebolt' API to 'launch app with search intent int source'
-        Then 'Firebolt' platform responds to '1st party app' with 'invalid parameters for discovery launch'
\ No newline at end of file
+        Then 'Firebolt' platform responds to '1st party app' with 'invalid parameters for discovery launch'
+
+    @DiscoveryLaunch @coreSDK @sdk @transport
+    Scenario: Discovery.Launch Hot Launch - Positive Scenario: Event validation where one app is in background and one is in foreground
+        Given the environment has been set up for 'DiscoveryLaunch' tests
+        And 3rd party 'certification' app is launched with 'secondary 3rd party app' appId
+        When Test runner waits for 10 'seconds'
+        And 'secondary 3rd party app' registers for the 'discovery onNavigateTo' event using the 'Firebolt' API
+        And 3rd party 'certification' app is launched
+        And '3rd party app' registers for the 'discovery onNavigateTo' event using the 'Firebolt' API
+        When 1st party app invokes the 'Firebolt' API to 'launch app with home intent'
+        Then 'Firebolt' platform responds to '1st party app' with 'true for discoverylaunch'
+        And '3rd party app' invokes the 'Firebolt' API to 'fetch lifecycle state'
+        Then 'Firebolt' platform responds with 'foreground for lifecycle state'
+        And 'Firebolt' platform triggers event 'onNavigateTo with home intent'
+        And 'Firebolt' platform triggers to 'secondary 3rd party app' event 'onNavigateTo with null response'
+
```

Comment:
the last line 229 should be below right ?
**And 'Firebolt' platform does not trigger to 'secondary 3rd party app' event 'onNavigateTo'**
You can confirm by testing in FC device

Reply:
the app in the background should not get the event. Only app in foreground should get the event

Reply:
also , what is the difference between the these 2 test cases ? Where is the test step which transitions the second app to inactive or background ?

Reply:
Updated the test step to "does not trigger" line as mentioned above.

Added the app transition step to background